src/wallet/wallet.cpp

```diff
@@ -145,7 +147,7 @@ void UnloadWallet(std::shared_ptr<CWallet>&& wallet)
     }
 }
 
-std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const WalletLocation& location, bilingual_str& error, std::vector<bilingual_str>& warnings)
+static std::shared_ptr<CWallet> LoadWalletInternal(interfaces::Chain& chain, const WalletLocation& location, bilingual_str& error, std::vector<bilingual_str>& warnings)
```

Comment:
nit: It seems using an unnamed namespace is preferable:
- https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rs-unnamed2
- https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions
- https://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static
- https://stackoverflow.com/questions/4977252/why-an-unnamed-namespace-is-a-superior-alternative-to-static

From the recen
- https://github.com/bitcoin/bitcoin/pull/19176#discussion_r435937252
- https://github.com/bitcoin/bitcoin/pull/19176#issuecomment-641304921
>  - Use anonymous namespace instead of static

Reply:
I think this is better for a follow up - being a bugfix we should make it minimal so it's easily backported. Unless we allow an anonymous namespace between functions?

Reply:
> I think this is better for a follow up - being a bugfix we should make it minimal so it's easily backported.

It's up to you :)

> Unless we allow an anonymous namespace between functions?

Why not?

Reply:
Placed in anonymous namespace.